src/components/section-panel/section-panel-header/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { Severity } from "@fellesdatakatalog/alert";
+import React, { FC } from "react";
+import { localization } from "../../../utils/language/localization";
+import MandatoryTag from "../../tag/mandatory-tag";
+import OptionalTag from "../../tag/optional-tag";
+import SC from './styled';
+
+interface Props {
+  sectionTitle: string;
+  dcatKeyword?: string;
+  alert?: Severity;
```

Comment:
Hvorfor bruker du Severity? Ser ut du kun bruker ERROR og derfor kan du bruker boolean hasError for eks.